evm/rpc/format.py

```diff
@@ -38,3 +40,24 @@ def block_to_dict(block, chain, include_transactions):
         block_dict['transactions'] = [encode_hex(tx.hash) for tx in block.transactions]
 
     return block_dict
+
+
+def format_params(*formatters):
+    def decorator(func):
+        @functools.wraps(func)
+        def formatted_func(self, *args):
+            assert len(formatters) == len(args), "could not apply %d formatters to %r" % (
```

Comment:
Assertion errors can be disabled.  Can we change to an explicit raise of something like `TypeError`?